passage/src/main/java/id/passage/android/PassageAliases.kt

```diff
@@ -2,23 +2,28 @@
 
 package id.passage.android
 
+import id.passage.android.api.CurrentuserAPI
 import id.passage.android.model.App
-import id.passage.android.model.AuthResult
 import id.passage.android.model.Credential
+import id.passage.android.model.CurrentUser
+import id.passage.android.model.MagicLink
 import id.passage.android.model.OAuth2ConnectionType
-import id.passage.android.model.OneTimePasscodeResponse
+import id.passage.android.model.User
+import id.passage.android.model.UserMetadataResponse
 
-public typealias PassageAuthResult = AuthResult
+public typealias PassageAppInfo = App
 
-public typealias OneTimePasscode = OneTimePasscodeResponse
+public typealias MagicLinkResponse = MagicLink
 
-@Deprecated("Use PassageAppInfo instead.", replaceWith = ReplaceWith("PassageAppInfo"))
-public typealias PassageApp = App
+public typealias CurrentUserInfo = CurrentUser
 
-public typealias PassageAppInfo = App
+public typealias PassagePasskey = PassageCredential
```

Comment:
Isn't `PassageCredential` just an unused alias? I think we can remove `PassageCredential`